parsel/selector.py

```diff
@@ -471,11 +508,14 @@ def get(self) -> str:
         Percent encoded content is unquoted.
         """
         try:
-            return etree.tostring(
-                self.root,
-                method=self._tostring_method,
-                encoding="unicode",
-                with_tail=False,
+            return typing.cast(
+                str,
+                etree.tostring(
```

Comment:
is there a type annoation missing in lxml types?

Reply:
`tostring` can return `bytes`, though now I see there is an overload handling `encoding="unicode"`, I'll check why it wasn't applied.

Reply:
Fixed by telling it that method= is not an arbitrary string.